### ISSUE TITLE
Some folders are missing during startup

### ISSUE THREAD

**Issue by f1yegor** [yandex/ClickHouse#131]
Could you fix and create if missing following folders

```
sudo mkdir -p data/default data/system
sudo mkdir -p metadata/default metadata/system
```

Error message: 

```
<Error> Application: File not found: /opt/clickhouse/metadata
```

Shouldn't be relevant, but I'm using docker image yandex/clickhouse-server:1.1.54023

**Comment by alexey-milovidov** [comment]
Yes, I will add creation of directories for default database at startup.

**Comment by f1yegor** [comment]
Fixed at 1.1.54030
